msal_extensions/libsecret.py

```diff
@@ -134,7 +130,5 @@ def trial_run():
   you may need to install gnome-keyring package.
 * Headless mode (such as in an ssh session) is not supported.
 """
-        logger.exception(message)  # This log contains trace stack for debugging
-        logger.warning(message)  # This is visible by default
-        raise
+        raise RuntimeError(message)  # Message via exception rather than log
```

Comment:
@chlowell , there are 3 snippets changed in this PR.

* In the first one, the original exception was an `ImportError`, and this PR would also throw an `ImportError`, so that is fine.
* In the second one, the original exception would be `ValueError` or `ImportError`, but it seems good enough for this PR to "normalize" them into - and raise - an `ImportError`.
* The third snippet here, this PR tentatively uses a generic `RuntimeError`. Do you think it would be beneficial to raise a customized `MsalExtensionsRuntimeError` instead?

Reply:
I think `RuntimeError` is reasonable, being the built-in type that seems most appropriate to this case. A custom exception is reasonable too but I lean toward the built-in for two reasons:
1. the value of a custom type is in enabling an application to choose a response suited to a specific failure case. This method doesn't know the specific failure case and it's irrelevant anyway because there's nothing a running application can do to make `LibsecretPersistence` work when this method fails.
2. you can change the type later without breaking anyone: `MsalExtensionsRuntimeError(RuntimeError)`